src/main/resources/spring/root-context.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+       xmlns:context="http://www.springframework.org/schema/context"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:jdbc="http://www.springframework.org/schema/jdbc"
+       xsi:schemaLocation="http://www.springframework.org/schema/beans
+        http://www.springframework.org/schema/beans/spring-beans.xsd
+        http://www.springframework.org/schema/context
+        http://www.springframework.org/schema/context/spring-context.xsd http://www.springframework.org/schema/jdbc http://www.springframework.org/schema/jdbc/spring-jdbc.xsd">
+
+    <import resource="query-context.xml"/>
+
+    <context:component-scan base-package="com.luxoft.svinenkov.movieland">
+        <context:exclude-filter type="regex" expression="com\.luxoft\.svinenkov\.movieland\.controller\..*"/>
+    </context:component-scan>
+
+    <bean id="jdbcTemplate" class="org.springframework.jdbc.core.JdbcTemplate">
+        <property name="dataSource" ref="dataSource"/>
+    </bean>
+
+    <bean id="namedJdbcTemplate" class="org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate">
+        <constructor-arg ref="dataSource"/>
+    </bean>
+
+
+    <!--<jdbc:embedded-database id="dataSource" type="HSQL">-->
+    <!--<jdbc:script location="classpath:/db/create-db.sql"/>-->
+    <!--<jdbc:script location="classpath:db/insert-cities.sql"/>-->
+    <!--</jdbc:embedded-database>-->
+
+    <import resource="query-context.xml"/>
+    <context:property-placeholder location="classpath:/property/app.properties"/>
+
+    <!--<bean id="dataSource" class="org.apache.commons.dbcp.BasicDataSource" destroy-method="close">-->
+        <!--<property name="driverClassName" value="${jdbc.driverClassName}"/>-->
+        <!--<property name="url" value="${jdbc.url}"/>-->
+        <!--<property name="username" value="${jdbc.username}"/>-->
+        <!--<property name="password" value="${jdbc.password}"/>-->
+    <!--</bean>-->
+
+    <bean id="dataSource" class="org.springframework.jdbc.datasource.DriverManagerDataSource">
+        <property name="driverClassName" value="${jdbc.driverClassName}"/>
+        <property name="url" value="${jdbc.url}"/>
+        <property name="username" value="${jdbc.username}"/>
+        <property name="password" value="${jdbc.password}"/>
+    </bean>
+
+
+    <!--<bean id="ds" class="org.apache.commons.dbcp.BasicDataSource">-->
```

Comment:
preferable use this instead of current - it has pool

Reply:
changed to "org.apache.commons.dbcp2.BasicDataSource"